src/pages/AddSongForm.js

```diff
@@ -78,10 +87,15 @@ function AddSongForm(props) {
                 type="text"
                 name="search"
                 id="search"
+                class="search_bar"
                 value={query.val}
                 onChange={handleChange}/>
-            <input type="button" value="Search" onClick={submitForm} />
-            <ResultsHeader id="resultsHeader"/>
+            <input type="button" value="Search" className="button_playlist" onClick={submitForm} />
+            <br></br><br></br>
+            { (afterSearch === false) ? (
+                <ResultsHeader id="resultsHeader"/>
+                ) : (<div/>
+            )}

```

Comment:
good, clean solution to only showing this element when afterSearch is false

Reply:
I agree. I was impressed when this efficient solution actually worked. Looks great on our website too.